### PR TITLE
Create proc_creation_win_lolbin_setres.yml

### PR DESCRIPTION
**Pull request by danielgottt**

### CODE REVIEW

**Comment by frack113** [comment]
Hi,
for me the detection is strange "choice" with setres as parent

File: rules/windows/process_creation/proc_creation_win_lolbin_setres.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+title: Use of Setres.exe
+id: 835e75bf-4bfd-47a4-b8a6-b766cac8bcb7
+status: experimental
+description: Detects the use of Setres.exe to set the screen resolution and then to potentially launch a file named "choice" from the local path
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_lolbin_setres.yml:4]:
```suggestion
description: Detects the use of Setres.exe to set the screen resolution and then potentially launch a file named "choice" (with any executable extension such as ".cmd" or ".exe") from the current execution path
```

File: rules/windows/process_creation/proc_creation_win_lolbin_setres.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    - attack.defense_evasion
+    - attack.t1218
+    - attack.t1202
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+      ParentImage|endswith: '\setres.exe'
+      Image|endswith: '\choice'
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_lolbin_setres.yml:22]: Indentation should be 4 not 2

**Review by nasbench — APPROVED** [review]
Approved after changes

**Pull request merged**